index.js

```diff
@@ -82,19 +83,20 @@ async function run() {
   // Join the room
   await callFrame.join({
     url: room,
+    // Comment out the above and uncomment the below, if you hard-coded a room for local testing (line 24)
+    // url: room.url,
     showLeaveButton: true,
   });
 
   // Leave handler
   function leave(e) {
     showEvent(e);
     callFrame.destroy();
-    document.getElementsByClassName('start-button')[0].style.display =
-      'initial';
-  }
+    document.getElementById('header').style.visibility = 'visible';
 
-  // Once a call starts running, hide the start button and prompts
-  document.getElementsByClassName('start-button')[0].style.display = 'none';
+    document.getElementsByClassName('page-text')[0].innerHTML =
```

Comment:
let's switch this to id so it's consistent with the other demo

Reply:
I promise I tried to do this! I left this comment in the PR and will keep testing: 

> Finally, I noticed something funny where my id for that "page-text" element disappeared as soon as the callFrame joined. I'm using getElementsByClassName[0] again there, because the class never disappears. If you have any tips for how I can avoid that and get the id to remain, I'm all ears.

Reply:
Ah ok no worries